src/ecs/commands/runCommandInContainer.ts

```diff
@@ -58,25 +61,12 @@ export async function runCommandInContainer(
         )
         const args = [JSON.stringify(execCommand.session), node.ecs.regionCode, 'StartSession']
         showOutputMessage(
-            `${moment().format(LOCALIZED_DATE_FORMAT)}  Container: "${node.containerName}" Task ID: "${
+            `${moment().format('YYYY-MM-DD HH:mm:ss:')}  Container: "${node.containerName}" Task ID: "${
```

Comment:
Hardcoding this isn't the right approach, as `LOCALIZED_DATE_FORMAT` hints.

Could start with `INSIGHTS_TIMESTAMP_FORMAT` (found with `git grep`) or create a new const. But there is already https://en.wikipedia.org/wiki/ISO_8601, and it looks like moment.js (the library we are using) has things to choose from: https://momentjs.com/docs/#/parsing/string-format/